config/application.rb

```diff
@@ -1,6 +1,7 @@
 require_relative 'boot'
 
 require 'rails/all'
+require_relative "../lib/open_food_network/i18n_config"
```

Comment:
this shouldn't be necessary, Rails should load code in `lib/`

Reply:
AFAIK `lib/` is not loaded by deafult only code within `app/`

Reply:
I found that this was necessary.

Reply:
We could auto-load lib, but is that the cleanest way to do it? It would mean that we load e.g. rake tasks into our main application. I would like a second opinion before doing this.

Reply:
AFAIK auto-loading `lib/` is not a good practice. I don't see it a big deal to add a require when we need a lib.

Reply:
`require_relative` is a bit of an anti-pattern in Rails, but we can live with it

Reply:
A tried to use `Rails.root`, but that was empty when loading a rake task.

Reply:
we should always try to use `require` but don't worry, let's move on. we'll see if it cause any problem in the future

:+1: